### DISCUSSION TITLE
How to couple Emax PD model to PBPK model

### DISCUSSION THREAD

**Discussion by humblewarrior** [Open-Systems-Pharmacology/Forum#788] [Questions & Problems] [answered]
Hi Everyone,
I really hope someone can help me here. I have built PBPK model for a small lipophilic molecule in PK-Sim. Separately, in NONMEM, I have also built a simple Emax model relating plasma concentrations of that molecule to increase in cardiovascular effects like heart rate and blood pressure.

Now, what I want to do is to link PBPK model to PD model in Mobi and simulate different dosing scenarios where I can look into how plasma concentrations are affecting cardiac concentrations and consequently driving heart rate. I understand that I have to first define the Emax equation in Mobi and that's where I am having trouble right now. I know there is a tutorial but its not been helpful to me so far. 
So there are 4 parameters in Emax equation:
1) Emax (which is dependent on plasma concentration)
2) Plasma concentration (which is time-variant)
3) EC50 (which is time-invariant)
4) Effect 

Obviously, Effect = Emax * plasma concentration /(plasma concentration +EC50)

Now, I have defined EC50 which is a constant and time-invariant. That is easy. Here are the next steps I am stuck at:
1) How to define effect and Emax and its units which is beats per minute for heart rate?
2) How about plasma concentration? How to put that into the equation as time-variant?

I am also willing to set up a small zoom and skype meeting and I really hope that someone can come to my rescue.

Regards,
Sumeet

**Comment by msevestre** [comment]
@humblewarrior if you want anyone to answer your questions, you **really** need to post them in the forum. This is the only place dedicated for interactions with other users. I will move this issue for you to the forum. Next time, the issue will be closed immediately

**Comment by Ricardo-DO** [comment, marked as answer]
Hi Sumeet

I don't know if when you say that the tutorial hasn't been helpful, you refer to this paper.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5080648/

The supplemental information is a step by step procedure of how to set up a PD model in MoBi.
The example is an IC50 but the way you set the equations should be pretty much the same to your EC50.
I am not sure if MoBi has beats per minute as units but I think that you can use "Count" unitless and have it as beat per minute.

Regards,
Ricardo

**Comment by humblewarrior** [comment]
Ricardo,

I appreciate your response.
Yes, I was referring to that same tutorial that your posted.
I will have to look into your recommendation of having "counts" use for
heart rate. I will try it this weekend and see how it turns out. Meanwhile,
any other recommendations will be much appreciated, too. Thanks!

Sumeet

**Comment by Slow-Walker66** [comment]
Hi Sumeet

I am learning Emax model in MoBi recently too, I have the same trouble in understanding how to define ODE formula of the plasma concentration.

As the tutorial said, the dCIP/dt = -log(2)/tlag*CIP, why there was log(2), and if my drug without a tlag, how could I construct the ODE formula?

Looking forward to hearing from you!

Xin

**Comment by ytg922** [comment]
I also have the same confusion, if Dr.Sumeet gain the understanding of the latest, please communicate with us.
